config.json

```diff
@@ -7,14 +7,30 @@
         },
         "idHash": {
             "algo": "sha256",
-            "hashLength": 24,
+            "bytes": 24,
             "iterations": 10000,
             "salt": ""
         },
         "initiateLifetime": {
             "hours": 1
         },
+        "loginDir": "/login/",
+        "loginIdLength": 24,
+        "loginLifetime": {
+            "minutes": 15
+        },
+        "challenge": {
+            "algo": "sha512",
+            "saltLength": 128
+        },
         "passwordSaltLength": 128,
+        "secureHash": {
+            "algo": "sha512",
+            "bytes": 48,
+            "derivation": "pbkdf2",
+            "iterations": 100000,
+            "saltLength": 64
```

Comment:
What is the difference between saltLength and bytes?  These two values are a bit confusing to me.